<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Camera</class>
 <widget class="QMainWindow" name="Camera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Switch">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1211</width>
      <height>671</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Coins">
     <attribute name="title">
      <string>硬币识别</string>
     </attribute>
     <widget class="QGroupBox" name="Photograph">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>10</y>
        <width>261</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>抓拍设置</string>
      </property>
      <widget class="QPushButton" name="snap">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>抓拍</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>路径</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="showRoad">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="road">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>格式</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="JPG">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>JPG</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="PNG">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>PNG</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="BMP">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>80</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>BMP</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RAW">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>80</y>
         <width>110</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>RAW</string>
       </property>
      </widget>
      <widget class="QPushButton" name="save_Picture">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="button_button">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>410</y>
        <width>261</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>按钮</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>98</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="Identify">
          <property name="text">
           <string> 识别</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="opencamera">
          <property name="text">
           <string>打开摄像头</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closecamera">
          <property name="text">
           <string> 关闭摄像头</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Camera_parameter">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>220</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>相机参数</string>
      </property>
      <widget class="QSlider" name="gain">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>51</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="expose">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>81</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>增益</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>曝光</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="gain_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>63</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="expose2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>80</y>
         <width>63</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Video_stream">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>461</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>图像显示</string>
      </property>
      <widget class="QGraphicsView" name="VIDEO1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>441</width>
         <height>491</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="PICTURE">
      <property name="geometry">
       <rect>
        <x>469</x>
        <y>9</y>
        <width>461</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>抓拍照片</string>
      </property>
      <widget class="QGraphicsView" name="picture1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>441</width>
         <height>491</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Prompt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>1191</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>文字提示</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>1171</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="Text">
          <property name="text">
           <string>Text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Text1">
          <property name="text">
           <string>Text1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Text2">
          <property name="text">
           <string>Text2</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Environment">
     <attribute name="title">
      <string>随机识别</string>
     </attribute>
     <widget class="QGroupBox" name="Photograph2">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>10</y>
        <width>261</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>抓拍设置</string>
      </property>
      <widget class="QPushButton" name="snap_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>抓拍</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>路径</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="showRoad_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="road_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>格式</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="JPG_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>JPG</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="PNG_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>PNG</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="BMP_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>80</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>BMP</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RAW_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>80</y>
         <width>110</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>RAW</string>
       </property>
      </widget>
      <widget class="QPushButton" name="save_Picture_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Camera_parameter_2">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>220</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>相机参数</string>
      </property>
      <widget class="QSlider" name="gain_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="expose_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>增益</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>曝光</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="gain_3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>63</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="expose4">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>80</y>
         <width>63</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="video_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>461</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>目标标注</string>
      </property>
      <widget class="QGraphicsView" name="VIDEO2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>441</width>
         <height>491</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="PICTURE2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>461</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>抓拍图片</string>
      </property>
      <widget class="QGraphicsView" name="picture2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>441</width>
         <height>491</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Prompt_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>1191</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>文字提示</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>19</y>
         <width>1171</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="Text_axis">
          <property name="text">
           <string>Text_axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Text4">
          <property name="text">
           <string>Text4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Text5">
          <property name="text">
           <string>Text5</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Button_button_2">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>410</y>
        <width>261</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>按钮</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="Scene_Identify">
          <property name="text">
           <string>开始识别</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close_camera">
          <property name="text">
           <string> 关闭摄像头</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCamera_Get">
    <property name="title">
     <string> </string>
    </property>
   </widget>
   <addaction name="menuCamera_Get"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
